src/App.vue

```diff
@@ -24,7 +24,7 @@
                                     </router-link>
                                 </li>
                                 <li class="nav-item mt-2 position-relative pl-2" v-if="canView && flag === 'beta'">
-                                    <router-link :to="authState.role === roles.dsa ? '/dsa/renewal?fromDate=&tab=nc&unContactedRenewalPrompters=true' : '/cashloan/renewal?fromDate=&tab=nc&unContactedRenewalPrompters=true'">
+                                    <router-link :to="authState.role === roles.cash_loan_agent ? '/cashloan/renewal?fromDate=&tab=nc&unContactedRenewalPrompters=true' : '/dsa/renewal?fromDate=&tab=nc&unContactedRenewalPrompters=true'">
```

Comment:
lets use computed to get this url, instead on repeating here

Reply:
done